### DISCUSSION TITLE
How does the OnFormFactor property work in Avalonia UI?

### DISCUSSION THREAD

**Discussion by al-zakaria** [AvaloniaUI/Avalonia#16685] [Q&A] [answered]
I'm writing a cross-platform application on Avalonia UI. and to better customize the look of the application on different platforms, I used the OnPlatform property, where each platform needs to be specified separately. But today I found a new property (I don't know how to call it correctly so I call it a property) called OnFormFactor where there are two parameters (I hope I called it right) Mobile and Desktop and I have a question. If I pass Desktop='16', Mobile='12' for conditional TextBlock in OnFormFactor property, what will happen in browser version? Will Avalonia customize it depending on the platform the browser is running on or something else?

**Comment by maxkatz6** [comment, marked as answer]
Yes, on browser it will depend on an actual device form factor.
Even more specifically, it will use https://developer.mozilla.org/en-US/docs/Web/API/NavigatorUAData/mobile

**Reply by al-zakaria** [reply]
Thank you!
